app/services/person_populator.rb

```diff
@@ -0,0 +1,47 @@
+class PersonPopulator
+  def initialize(person, profile)
+    @person = person
+    @profile = profile
+  end
+
+  def call
+    populate_standard_attributes
+    populate_identifier_attributes
+    person.save
+  end
+
+private
+
+  attr_reader :person, :profile
+
+  STANDARD_ATTRIBUTES = %w[
+    first_names
+    last_name
+    date_of_birth
+    gender_additional_information
+    latest_nomis_booking_id
```

Comment:
should not `latest_nomis_booking_id` remain to profile?

Reply:
See https://dsdmoj.atlassian.net/browse/P4-1435 :). AFAIK a person and a profile and a booking are three different things. latest_nomis_booking_id is the same as the other identifiers except for the fact that it doesn't represent a person in other systems. It references a booking. Probably having the identifiers on a person makes more sense so I agree with Jonny in deciding to move this here.

Reply:
Said another way - a person may have many bookings (times they were booked into prison)

Reply:
I had a chat with @jcavell , I was worried because we have the same field in the profile table, and that is confusing.
So we agreed on renaming the field in the profile (Jonny created a ticket for that)